sky/shell/platform_view.h

```diff
@@ -77,11 +77,14 @@ class PlatformView {
 
   std::unique_ptr<Rasterizer> rasterizer_;
   std::unique_ptr<Engine> engine_;
+  SkyEnginePtr engine_ptr_;
```

Comment:
My bad. I did not think about the Java side hosting the engine. It is not enough to get a hold of the engine however. The SkyEngine method overrides are private. We would have to make them public and ensure the threading restrictions are enforced.